lib/bootstrap_form/form_group.rb

```diff
@@ -50,15 +50,23 @@ def form_group_control_class(options)
     end
 
     def form_group_classes(options)
-      classes = ["form-group", options[:class].try(:split)].flatten.compact
-      classes << "row" if group_layout_horizontal?(options[:layout]) && classes.exclude?("form-row")
-      classes << "form-inline" if field_inline_override?(options[:layout])
+      classes = ["mb-3", options[:class].try(:split)].flatten.compact
+      classes << "row" if horizontal_group_with_gutters?(options[:layout], classes)
+      classes << "col-auto g-3" if field_inline_override?(options[:layout])
       classes << feedback_class if options[:icon]
       classes
     end
 
+    def horizontal_group_with_gutters?(layout, classes)
+      group_layout_horizontal?(layout) && !classes_include_gutters?(classes)
+    end
+
     def group_layout_horizontal?(layout)
       get_group_layout(layout) == :horizontal
     end
+
+    def classes_include_gutters?(classes)
+      classes.any? { |c| c =~ /g-\d+/ }
```

Comment:
Do you need to anchor this regex at the start or end of the string? I'm thinking about a case where the user is using a class name like `setting-5`. (I haven't even looked at the Bootstrap 5 classes yet, so please forgive me if this comment has an obvious answer.)

Reply:
Your concern makes total sense - and you're right, it should be anchored - I'd say both sides.